test-data/unit/check-flags.test

```diff
@@ -515,3 +515,59 @@ x, y = 1, 2  # type: Unchecked, Unchecked
 [out]
 main:4: error: Type of variable becomes "Any" due to an unfollowed import
 main:6: error: A type on this line becomes "Any" due to an unfollowed import
+
```

Comment:
[Done!](https://github.com/python/mypy/pull/3519/commits/52b9206294769d65c0b3b524c0d5429c23911ba1)

Let me know if you have any other test ideas.